kafka_consumer_freshness_tracker/README.md

```diff
@@ -47,7 +47,8 @@ bazel-bin/kafka_consumer_freshness_tracker/src/main/java/com/tesla/data/consumer
 
 Bundled as a jar you can run the Tracker with a configuration file with the `--conf <path to conf>` flag. For local 
 testing, you can also just add the `--once` flag to just run the Tracker once, and then dump the metrics to the console, 
-before stopping.
+before stopping. The `--strict` flag can be used to run the Tracker in strict mode, which enforces stricter validation of the cluster
```

Comment:
what does stricter actually mean though? It should describe expected behavior of the application with and without the flag. For example, part of that could be: "with `--strict` enabled, the application will crash if the bootstrap servers specified for the Tracker do not match the bootstrap servers used for configuring the burrow instance).

I'm interested in what behavior we are claiming to have here for users and then when I'm going through the review, i check to see if we (a) do what we say here and (b) add tests that check that as well

Reply:
Beefed this up, let me know if you feel anything's missing